src/components/Breadcrumbs/Breadcrumbs.vue

```diff
@@ -412,22 +427,16 @@ export default {
 			// Use a breadcrumb component for the hidden breadcrumbs
 			crumbs.push(createElement('Breadcrumb', {
 				class: 'dropdown',
-				// We want the Actions to always show as a dropdown menu
-				props: {
-					// Don't show a title for this breadcrumb, only the Actions menu
-					title: '',
-					forceMenu: true,
-					// Don't allow dropping directly on the actions breadcrumb
-					disableDrop: true,
-					hideable: false,
```

Comment:
@raimund-schluessler this seemed to not be usd anywhere, so I deleted it, I think it was a leftover :)

Reply:
Yes, this was a leftover. Thanks for taking care.